bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -245,7 +248,12 @@ def move_gripper_to_point(self, point, rotation):
 
         hand_pose = math_helpers.SE3Pose(*point, quat)
         hand_trajectory = trajectory_pb2.SE3Trajectory(
-            points=[trajectory_pb2.SE3TrajectoryPoint(pose=hand_pose.to_proto())]
+            points=[
```

Comment:
The method `move_gripper_to_point` now takes in the trajectory duration, and also executes the trajectory , returning the success status